src/main/java/Duke.java

```diff
@@ -1,10 +1,126 @@
+                + indent + indent + task
+                + System.lineSeparator()
+                + indent + "Now you have " + allTasks.size() + " tasks in the list.");
+    }
+
+    // mark a Task as done
+    public static void markTaskAsDone(int num) {
+        if (num > allTasks.size() || num == 0) {
+            System.out.println(indent + "Sorry, this task does not exist!");
+        }
+        else {
+            allTasks.get(num - 1).setIsDone(true);
+            System.out.println(indent + "Nice! I've marked this task as done:");
+            System.out.println(indent + "[\u2713] " + allTasks.get(num - 1).getTask());
+        }
+    }
+
+    public static void addTask(String[] inputArr) {
+
+        /* TODO: implement error checking for user input
+           cases:
+           - input is completely empty
+           - inputArr[0] == _todo/deadline/event, but rest of the input is completely empty
+           - deadline/event does not have "/by" or "/at"
+        */
+
+        Task newObj;
+
+        switch(inputArr[0]) {
+            case("todo"): {
+                String[] descArr = Arrays.copyOfRange(inputArr, 1, inputArr.length);
+                newObj = new Todo((convertToString(descArr)));
+                break;
+            }
+            case("deadline"): {
+                int pos = Arrays.asList(inputArr).indexOf("/by");
+                String[] descArr = Arrays.copyOfRange(inputArr, 1, pos);
+                String[] byArr = Arrays.copyOfRange(inputArr,pos+1,inputArr.length);
+                newObj = new Deadline(convertToString(descArr),convertToString(byArr));
+                break;
+            }
+            case("event"): {
+                int pos = Arrays.asList(inputArr).indexOf("/at");
+                String[] descArr = Arrays.copyOfRange(inputArr, 1, pos);
+                String[] atArr = Arrays.copyOfRange(inputArr,pos+1,inputArr.length);
+                newObj = new Event(convertToString(descArr),convertToString(atArr));
+                break;
+            }
+            default: {
+                newObj = new Task((convertToString(inputArr)));
+            }
+        }
+        allTasks.add(newObj);
+        printNewlyAddedTask(newObj);
+    }
+
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println(indent + "Hello! I'm Duke\n" + indent + "What can I do for you?\n");
+
+        Scanner line = new Scanner(System.in);
+        String input = line.nextLine();
+
+        String userCmd = "";
+
+        while(!input.toLowerCase().equals("bye")) {
+            String[] inputArr = input.split(" ");
+
+            userCmd = inputArr[0].toLowerCase();
+
+            switch (userCmd) {
```

Comment:
Should the case statements be indented to the same extent as the switch () statement?